operator/api/v1alpha1/securedcluster_types.go

```diff
@@ -226,6 +226,10 @@ type PerNodeSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=3,displayName="Node Scanning Settings"
 	NodeInventory *ContainerSpec `json:"nodeInventory,omitempty"`
 
+	// Settings for the SFA container, which is responsible for file activity monitoring on the Node.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=3,displayName="SFA"
+	SFA *ContainerSpec `json:"fact,omitempty"`
```

Comment:
I didn't review the entire PR just yet, but I'm not sure about adding this field to the CRD just yet.

This field only makes sense when the feature flag is enabled. I would remove the field as long as the feature is in dev preview, and if you need to modify the container resources during that time, you could do so using [overlays](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_security_for_kubernetes/4.4/html/installing/installing-rhacs-on-red-hat-openshift#customize-installation-operator-overlays_install-central-config-options-ocp)

Reply:
What are the downsides of having this field from the start? And how was it done before for other new components (e.g. compliance, node inventory)?